docker/minio/s3_entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Create empty buckets on every container startup
+# Note: $0 is included in the exec because "/bin/bash -c" swallows the first
+# argument, so it must be re-added at the beginning of the exec call
```

Comment:
This comment is no longer accurate, since we're no longer calling `/bin/bash -c` as part of the entrypoint. It can be removed.
```suggestion
```